File: SAMSUI/src/app/Model/asset/modelItem.ts
export class ModelItem{
    public  modelItemId ?:number;
    public  orgId?:number;
	public  modelId ?:number;
	public  itemMasterId ?:number;
    public 	itemMasterName?:string;
    public 	itemMasterDesc?:string;
    public 	itemTypeName?:string;
    public 	masterUOMName?:string;	
	public  active ?:string;
    public  activeDisp ?:boolean;
    public  createdBy?:string; 
	public  createdDt?:Date; 
	public  updatedBy?:string; 
    public  updatedDt?:Date; 
    public  updatedDtDisp?:string;
	public  createdDtDisp?:string;
}

File: sams-backend/src/main/java/com/sams/entity/ModelItem.java
package com.sams.entity;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import java.time.LocalDateTime;

@Entity
@Table(name = "model_item", schema = "asset")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ModelItem {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "model_item_id")
    private Long modelItemId;

    @Column(name = "org_id")
    private Long orgId;

    @Column(name = "model_id")
    private Long modelId;

    @Column(name = "item_id")
    private Long itemId;

    @Column(name = "active")
    private Boolean active;

    @Column(name = "created_by")
    private String createdBy;

    @Column(name = "created_dt")
    private LocalDateTime createdDt;

    @Column(name = "updated_by")
    private String updatedBy;

    @Column(name = "updated_dt")
    private LocalDateTime updatedDt;

}